### PR TITLE
Add Prettier code formatter 

### PR DESCRIPTION
**Pull request by PaulChinkwende**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-497

- [x] Jira ticket linked
- [x] PR linked on Jira ticket

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "es6": true,
     "browser": true,
     "jest": true,
-    "node": true
+    "node": true,
+  "extends": ["plugin:prettier/recommended"]
```
Comment (sareh) [inline .eslintrc]: You can add this prettier plugin to line 2.

Comment (PaulChinkwende) [reply .eslintrc]: 👍

File: package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/articles-renderer/issues"
+  },
+  "homepage": "https://github.com/bbc/articles-renderer#readme",
+  "dependencies": {
+    "@jaredpalmer/after": "^1.3.1",
+    "express": "^4.16.3",
+    "prop-types": "^15.6.1",
+    "prettier": "^1.12.1",
```
Comment (sareh) [inline package.json:30]: You should put prettier & eslint config/plugin modules in the devDependencies.

**Comment by sareh** [comment]
Closed since this PR supersedes it: https://github.com/bbc/articles-renderer/pull/8/files

**Pull request closed without merging**